jiant/models.py

```diff
@@ -525,6 +525,28 @@ def build_task_specific_modules(task, model, d_sent, d_emb, vocab, embedder, arg
     elif isinstance(task, EdgeProbingTask):
         module = EdgeClassifierModule(task, d_sent, task_params)
         setattr(model, "%s_mdl" % task.name, module)
+    elif isinstance(task, (MultiRCTask, ReCoRDTask)):
+        module = build_qa_module(task, d_sent, model.use_bert, task_params)
+        setattr(model, "%s_mdl" % task.name, module)
+    elif isinstance(task, CharSeq2SeqTask):
+        log.info("using {} attention".format(args.s2s["attention"]))
```

Comment:
probably want to use an assert that `args.char_embs` is turned on ?

Reply:
In the current implementation, args.char_embs should be switched off - this tasks treats chars as words, i.e., it uses a tokenizer that splits the input into individual chars.